.github/workflows/dev_pr/issue_check.js

```diff
@@ -78,11 +78,42 @@ async function commentNotStartedTicket(github, context, pullRequestNumber) {
     }
 }
 
+async function verifyGitHubIssue(github, context, pullRequestNumber, issueID) {
+    const issueInfo = await helpers.getGitHubInfo(github, context, issueID, pullRequestNumber);
+    if (issueInfo) {
+        if (!issueInfo.assignees.length) {
+            await github.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: pullRequestNumber,
+                body: ":warning: GitHub issue #" + issueID + " **has not been assigned in GitHub**, please assign the ticket."
```

Comment:
Should we just assign it to the person opening the PR? Unlike JIRA issues gh issues can have more than one assignee (up to 10), so that should not require any fancy checking.

Reply:
If the issue does not have any assignee it will be automatically assigned to the PR creator. We will still prompt a warning that this has been done.